tools/nugets.linq

```diff
@@ -11,6 +13,10 @@ void Main()
 	var samplesDirectory = Path.Combine(docsDirectory, "samples");
 	var tutorialsDirectory = Path.Combine(docsDirectory, "tutorials");
 	var nugetConfigFile = Path.Combine(docsDirectory, "nuget.config");
+	
+	var filter = Interaction.InputBox("To filter solutions to update, enter a simple filter value. Leave blank to update all solutions.",
```

Comment:
And that's fine. I'm just saying there's something built in that linqpad users might be more familiar with.